java/ql/lib/change-notes/2022-09-08-implicit-read-flowstates.md

```diff
@@ -0,0 +1,4 @@
+---
+category: fix
+---
+* Fixed an issue in the taint tracking analysis where implicit reads were not allowed by default in sinks or additional taint steps that used flow states.
```

Comment:
If we think this needs a change note, then it probably needs to be for all the languages.

Reply:
Do you think the change note isn't needed?

Reply:
I could be convinced either way - no strong feelings. It's fairly technical. Argument in favour: If someone is writing queries using flow state and had trouble with this, then it could be relevant. Argument against: If you're mostly just running queries out-of-the-box then it's probably not that likely to affect you, and as-is it's likely hard to understand from that perspective.

Reply:
Let's risk being verbose: 6f9b856.